change/@internal-react-composites-2ec9225a-d4e0-42e1-a975-3a987736beb3.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "implemented custom datamodel functionality to meeting composite excluding sidebar.",
```

Comment:
Nitty nit: Capitalize line (Implemented...), and no trailing period.
Also, we're referring to sidebase as "side pane".

Reply:
From the PR, it looks like this is only implementing it for the side pane (not excluding it)?

Reply:
this is extending the existing functionality of the calling and chat composite to accept the prop from the meeting composite level. the People pane part of the side pane still needs done.